docs/site/index/cldr-spec/coverage-levels.md

```diff
@@ -32,7 +32,7 @@ To filter "at that level or above", you use the fact that basic ⊂ moderate ⊂
 
 ### Migration
 
-As of v43, the files in **/seed/** have been moved to **/common/**. Older versions of CLDR separated some locale files into a 'seed' directory. Some implementations used for filtering, but the criteria for moving from seed to common were not rigorous. To maintain compatibility with the set of locales used from previous versions, an implementation may use the above process for Basic and above, but then also add locales that were previously included. For more information, see [CLDR 43 Release Note](/index/downloads/cldr-43). 
+As of v43, the files in **/seed/** have been moved to **/common/**. Older versions of CLDR separated some locale files into a 'seed' directory. Some implementations used for filtering, but the criteria for moving from seed to common were not rigorous. To maintain compatibility with the set of locales used from previous versions, an implementation may use the above process for Basic and above, but then also add locales that were previously included. For more information, see [CLDR 43 Release Note](https://cldr.unicode.org/downloads/cldr-43). 
```

Comment:
it's better to use relative links. You can just remove the /index

so it would be (/downloads/cldr-43)

example of a relative link elsewhere on the site: https://github.com/unicode-org/cldr/blob/main/docs/site/translation.md?plain=1#L26

Reply:
fixed, I think